samples/sample.yaml

```diff
@@ -26,5 +26,13 @@ reports:
       - name: "in-progress"
         config:
           report-on: "Epic"
-          wip-label-match: "(\\d+)-dev"   
+          wip-label-match: "(\\d+)-dev"
+      - name: "issue-count-cycle-time"
+        config:
+          feature-cycletime-limit: 200
+          epic-cycletime-limit: 200
+          documentation-cycletime-limit: 200
+          report-on: ["feature", "epic", "documentation" ]
+          wip-label-match: "(\\d+)-dev"
```

Comment:
e.g. in defaults this would be (\\d+)-wip with it being overriden here with -dev ... all the rest can be defaults so this is likely the only line that survives in the yaml file